notifications.yaml

```diff
@@ -0,0 +1,268 @@
+                Location (URL) of accepted message resource.
+
+                A GET request to this URL returns the message status and
+                details.
+definitions:
+  ProblemJson:
+    title: Problem Type
+    type: object
+    properties:
+      type:
+        type: string
+        format: uri
+        description: |
+          An absolute URI that identifies the problem type.  When dereferenced,
+          it SHOULD provide human-readable documentation for the problem type
+          (e.g., using HTML).
+        default: 'about:blank'
+        example: 'https://example.com/problem/constraint-violation'
+      title:
+        type: string
+        description: |
+          A short, summary of the problem type. Written in english and readable
+          for engineers (usually not suited for non technical stakeholders and
+          not localized); example: Service Unavailable
+      status:
+        type: integer
+        format: int32
+        description: >
+          The HTTP status code generated by the origin server for this
+          occurrence
+
+          of the problem.
+        minimum: 100
+        maximum: 600
+        exclusiveMaximum: true
+        example: 503
+      detail:
+        type: string
+        description: |
+          A human readable explanation specific to this occurrence of the
+          problem.
+        example: Connection to database timed out
+      instance:
+        type: string
+        format: uri
+        description: >-
+          An absolute URI that identifies the specific occurrence of the
+          problem.
+
+          It may or may not yield further information if dereferenced.
+  MessageContent:
+    title: MessageContent
+    type: object
+    properties:
+      body_short:
+        type: string
+        description: Message content (short version).
+        minLength: 3
+        maxLength: 100
+      body_long:
+        type: string
+        description: Message content (long version).
+        minLength: 3
+    required:
+      - body_short
+  MessageDefinition:
+    title: MessageDefinition
+    type: object
+    properties:
+      to:
+        description: The fiscal code of the destination user.
+        type: string
+      dry_run:
+        type: boolean
+        description: >-
+          This parameter, when set to true, allows developers to test a request
+          without actually sending a message.
+
+
+          The default value is false.
+      time_to_live:
+        type: integer
+        description: >-
+          This parameter specifies how long (in seconds) the message should be
+          kept in storage if the device is offline. The maximum time to live
+          supported is 1 year, and the default value is 1 hour.
+        minimum: 3600
+        maximum: 31536000
+      content:
+        $ref: '#/definitions/MessageContent'
+    required:
+      - to
+      - content
+  MessageStatus:
+    title: MessageStatus
+    type: object
+    properties:
```

Comment:
I see, the confusion for me is that the `ACCEPTED` state is provided for each channel, but the `accepted_at` is for the whole message - something is semantically not right

also another issue I'm thinking of is: why the sender, that cannot decide what channels the message should be sent to, may want to know the status of delivery for each channel? and actually has to check the status for each channel? at the end what he wants to know is whether the message has been delivered to the user through at least a channel (the fact that may be more than one channel is actually purposely hidden from him)

so I think that we should not expose the state of each channel, but just tell the sender whether the message as been delivered to at least one channel

also, how is the information that some channel succeeded but some other channel failed useful to the sender? if he attempts to retry the delivery by sending a new message, given that he cannot limit the channels to the failed ones, he would send duplicate messages!!!

so I don't see how is actually useful to provide the status information for each channel